index.js

```diff
@@ -52,7 +52,9 @@ const D2LHypermediaRels = {
 		previousPeriod: 'https://activities.api.brightspace.com/rels/previous-period',
 		previousWeek: 'https://activities.api.brightspace.com/rels/previous-week',
 		feedback: 'https://activities.api.brightspace.com/rels/feedback',
-		activityGrade: 'https://activities.api.brightspace.com/rels/activity-grade'
+		activityGrade: 'https://activities.api.brightspace.com/rels/activity-grade',
+		childActivityUsage: 'https://activities.api.brightspace.com/rels/child-activity-usage',
+		childUserActivityUsage: 'https://activities.api.brightspace.com/rels/child-user-activity-usage'
```

Comment:
is this child as in parent/child, or child like information structurally?

Reply:
We call it several things, but it's eg. the underlying/linked activity usage for a content topic activity usage (should it have one).

Reply:
Amusing anecdote, I've always considered the activity to be a child of the underlying domain-activity 🤷‍♂️ Good to settle on a term though, whatever it may be!